pkg/storaged/pages.jsx

```diff
@@ -816,14 +816,14 @@
     return (
         <Page id="storage">
             { parent_crumbs.length > 0 &&
-            <PageBreadcrumb stickyOnBreakpoint={{ default: "top" }}>
+            <PageBreadcrumb {...(!client.in_anaconda_mode() ? { stickyOnBreakpoint: { default: "top" } } : {})}>
                 <Breadcrumb>
                     { parent_crumbs }
                     <BreadcrumbItem isActive>{page_display_name(page)}</BreadcrumbItem>
                 </Breadcrumb>
             </PageBreadcrumb>
             }
-            <PageSection isFilled={false}>
+            <PageSection isFilled={false} padding={{ default: !client.in_anaconda_mode() > 0 ? "padding" : "noPadding" }}>
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19876-20240123-135148-fddbc75a-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)